agent/06_agent_create_cluster.sh

```diff
@@ -219,6 +219,13 @@ function wait_for_cluster_ready() {
       exit 1
   fi
 
+  if [[ $AGENT_TEST_CASES =~ "skip_wait_for_install_complete" ]]; then
```

Comment:
Would it make sense to set this automatically when the platform=`vsphere`?

Reply:
That's what I had it set to originally :). But then though to have it independent of platform type in the event we eventually fully support the vsphere environment. I'm ok either way.

Reply:
OK that's fine leaving it as independent. I remembered that external platform can probably use this too.

Reply:
TBTH this doesn't seem to me a real test case: it doesn't exercise the environment nor perform any verification, and it's not used in `run_agent_test_cases` (and IIUC we're not even performing a complete installation). So, as long as it is useful to verify a part of the agent vsphere workflow, it could be managed via a normal config var, ie `WAIT_FOR_INSTALL_COMPLETE=false`

Reply:
Thanks Andrea. I've switch to using a normal config var WAIT_FOR_INSTALL_COMPLETE as you suggested.